app/routes/index.tsx

```diff
@@ -36,11 +37,14 @@ export default function Index() {
           <h2>{landing.title}</h2>
           <p className="text">{landing.subtitle}</p>
           <p className="text"></p>
-          <Form action="/api/auth/auth0" method="post">
-            <button className="button" name="login" type="submit" value="true">
-              {landing.getStarted}
-            </button>
-          </Form>
+          <button
+            className="button"
+            name="login"
+            value="true"
+            onClick={() => fetcher.load("/api/auth/auth0")}
```

Comment:
I'm curious as to why you seem to like this pattern better than using the <Form>. In essence they do very similar things, but not the same. 

Form, hits the server. And because of that, Form works without JS.
fetcher.load, is a client side api call, and thus requires JS to work.

Reply:
Yes, I agree; this is something that I want to check; using the Form to redirect to Auth0 is crashing the web app, and we need to keep more into this to find a more solid solution.